emmet/vasp/settings/materials_settings.json

```diff
@@ -348,7 +469,8 @@
   {
     "materials_key": "scan_structure",
```

Comment:
Instead of `scan_structure`, would it make sense to just store `ComputedStructureEntry` instead of `ComputedEntry` in the `.entries` field? I can imagine users wanting to be able to query each structure, even though they will match within a tolerance

Reply:
For now I have retained `scan_structure` and also added `r2scan_structure` and `pbesol_structure` keys